internal/backend/sessions/sessionworkflows/modules/akmodules.go

```diff
@@ -0,0 +1,18 @@
+package modules
+
+import (
+	"go.autokitteh.dev/autokitteh/internal/backend/fixtures"
+	"go.autokitteh.dev/autokitteh/internal/backend/sessions/sessionworkflows/modules/os"
+	"go.autokitteh.dev/autokitteh/internal/backend/sessions/sessionworkflows/modules/store"
+	"go.autokitteh.dev/autokitteh/internal/backend/sessions/sessionworkflows/modules/time"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+func IsAKModuleExecutorID(xid sdktypes.ExecutorID) bool {
+	switch xid {
+	case store.ExecutorID, time.ExecutorID, os.ExecutorID, fixtures.ModuleExecutorID:
+		return true
+	default:
+		return false
```

Comment:
Can be outside the `switch`

Reply:
not sure what you mean - the `return false`? what for?